src/components/content-header-card/content-header-card.module.css

```diff
@@ -0,0 +1,174 @@
+.contentHeaderCard {
+	padding: 0;
+
+	& .cardTop {
+		padding: 16px;
+		display: flex;
+
+		& .icon {
+			margin-right: 12px;
+			display: none;
+
+			@media (--dev-dot-tablet-up) {
+				display: block;
+			}
+		}
+
+		& .content {
+			flex-grow: 1;
+
+			& .upper {
+				display: flex;
+				gap: 16px;
+				flex-direction: column-reverse;
+				@media (--dev-dot-tablet-up) {
+					flex-direction: row;
+				}
+
+				& .left {
+					flex-grow: 1;
+
+					& .titleWrap {
+						display: flex;
+						flex-wrap: wrap;
+						align-items: baseline;
+						@media (--dev-dot-tablet-up) {
+							flex-wrap: unset;
+						}
+
+						& .title {
+							color: var(--token-color-foreground-strong);
+						}
+
+						& .attribution {
+							color: var(--token-color-foreground-faint);
+							margin-left: 4px;
+						}
+					}
+
+					& .description {
+						margin-top: 2px;
+						color: var(--token-color-foreground-primary);
+					}
+				}
+
+				& .dropdown {
+					flex-shrink: 0;
+					width: 100%;
+					& > button {
+						width: 100%;
+					}
+
+					@media (--dev-dot-tablet-up) {
+						width: auto;
+						& > button {
+							width: auto;
+						}
+					}
+				}
+			}
+		}
+
+		& .lower {
+			display: flex;
+			flex-direction: column;
+			align-items: flex-start;
+			@media (--dev-dot-tablet-up) {
+				flex-direction: row;
+				align-items: flex-end;
+			}
+
+			& .badgeList {
+				margin-top: 16px;
+			}
+
+			& .note {
+				color: var(--token-color-foreground-faint);
+				flex-grow: 1;
+				text-align: right;
+				margin-top: 12px;
+				@media (--dev-dot-tablet-up) {
+					margin-top: 6px;
+				}
+			}
+		}
+	}
+
+	& .cardBottom {
+		padding: 12px 16px;
+		background: var(--token-color-surface-faint);
+
+		/* This is the color from --token-surface-base-box-shadow,
+		which the color value from it is not in a variable alone */
+		border-top: 1px solid rgba(101, 106, 118, 0.2);
+		border-radius: 0 0 6px 6px;
+		display: flex;
+		flex-direction: column;
+		align-items: flex-start;
+		gap: 16px;
+
+		@media (--dev-dot-tablet-up) {
+			flex-direction: row;
+			justify-content: space-between;
+			align-items: center;
+		}
+
+		& .linksList {
+			list-style: none;
+			display: flex;
+			padding: 0;
+			margin: 0;
+			& li {
+				&:not(:first-of-type) {
+					margin-left: 12px;
+				}
+				& > * {
+					/* TODO: Rather than override our existing link component, 
+					we should eject or build in support for what we're trying
+					to do here. These styles assume the StandaloneLink styles
+					are part of its contract, but they are not and are subject
+					to change */
+					text-decoration: underline;
+					color: var(--token-color-foreground-primary);
+					&:hover {
+						color: var(--token-color-foreground-strong);
+					}
```

Comment:
💡 Should we adjust standalone link to support this style variation?

Reply:
I don't think we need to add an additional variant in this case where just `text-decoration` and `color` are overridden. In both approaches we'd still be using `StandaloneLink`, so they have equal risk in being unintentionally affected by changes to the component. Since `StandaloneLink` is widely used, changes to it would need to be well-tested across references anyways, so using the component here doesn't increase maintenance effort from my perspective.

Reply:
When we've taken the "override" approach in the past, it's bitten us pretty badly down the road. Our transition to CSS modules was motivated in part by feeling that components were brittle -- we could no longer change the markup or styling with an acceptable degree of confidence (in a reasonable amount of time) because of having to account for a slew of widely varying "overrides" that had been layered on top of each component.

I'd agree that for a relatively atomic component like `StandaloneLink` this may be less of an issue. I also do agree though that we should avoid these overrides regardless - personally I think the `TODO` here feels accurate, we should either eject or build in support for this theme option.

Stepping back a bit, I am kind of curious as to the design system logic here - it looks like the main change here is we show an underline by default (the color difference compared to our `secondary` version of `StandaloneLink` seems subtle). Is that possibly a divergence from the [HDS Standalone Link](https://helios.hashicorp.design/components/link/standalone)? If so I think it might be worth checking with design on why that change feels necessary.